pom.xml

```diff
@@ -101,6 +102,7 @@
   <properties>
     <hawkular-apm-db>elasticsearch</hawkular-apm-db>
 
+    <plugin.version.com.spotify>0.4.9</plugin.version.com.spotify>
```

Comment:
This should be named as `version.com.spotify-docker-maven-plugin`. And the shade plugin too, btw that can be consumed from parent pom.

https://github.com/hawkular/hawkular-parent-pom/blob/master/pom.xml#L186